xarray/core/concat.py

```diff
@@ -92,15 +93,20 @@ def concat(
         - 'override': if indexes are of same size, rewrite indexes to be
           those of the first object with that dimension. Indexes for the same
           dimension must have the same size in all objects.
+    combine_attrs : {'drop', 'identical', 'no_conflicts', 'override'}, optional
```

Comment:
```suggestion
    combine_attrs : {'drop', 'identical', 'no_conflicts', 'override'}, default 'override'
```

Reply:
need a line break to avoid going over 79 characters
```suggestion
    combine_attrs : {'drop', 'identical', 'no_conflicts', 'override'},
                    default 'override
```